src/ILCompiler.TypeSystem/src/TypeSystem/Interop/IL/NativeStructType.Mangling.cs

```diff
@@ -0,0 +1,13 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Internal.TypeSystem.Interop
+{
+    public partial class NativeStructType : MetadataType, IPrefixMangledType
```

Comment:
Looks like the CoreFX coding guidelines say (via example) to strip the visibility and base class so we should standardize on that.